posts/2023/installing-farmos-on-ddev-with-composer/index.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Installing farmOS on DDEV with Composer
+date: 2023-05-17
+author: rgcarr
```

Comment:
Not essential (and I don't know how we'd do it with the front-matter), but I wonder if we want to have a pattern of including an "et al" link back to the PR here when the authorship of the post is complex...

Reply:
Yeah I was starting to wonder the same... Gastby might have a way to see & link to the commit authors for a given post? Though this might not always be ideal.

Reply:
Agreed. I tend to favor explicitly referencing the authors or the pull request in the front-matter rather than something automatic using git. I feel it introduces too many complexities when your build system starts inspecting the source control layer itself. (for more than just basic branch/release/tags)

Reply:
@wotnak would you like to be listed as an author here?